power-the-polls-form/Readme.md

```diff
@@ -0,0 +1,62 @@
+# Power the Polls
+
+This is a custom web component that allows our partners to embed the Power the Polls signup form on their own web sites and intranets. Contact info@powerthepolls.org to discuss any partnerships.
+
+## How to embed
+
+### HTML
+
+Include the latest version:
+
+```html
+  <script type="module" src="https://unpkg.com/@username/power-the-polls-form@1.0.0/power-the-polls-form.js"></script>
+```
+
+Reference it like any other HTML element:
+
+```html
+<power-the-polls-form
+  destination="http://optional-form-target-here"
+  custom-form-field-label="Label for your own custom field here"
```

Comment:
Should include a way to hard-code the source code

Reply:
Not sure I understand